GitIQT/Scenarios/CommitChange.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace GitIQT
+{
+    internal class CommitChange : IScenario
+    {
+        private string Response = string.Empty;
+
+        private string Answer = string.Empty;
+
+        // Ask user to commit changes to their branch
+        public void AskPrompt()
+        {
+            // Ask user to commit changes to their branch
+            Answer = $"git commit";
+
+            var prompt = $"What git command do you need to type in to commit changes to the branch? (git add has already been performed)";
```

Comment:
![image](https://user-images.githubusercontent.com/26447240/214761678-f7155059-f611-4b53-86e5-ee57f2cfb762.png)

You also need to do the commit message